autoresolution/src/plugin.py

```diff
@@ -104,6 +104,32 @@ def setDeinterlacer(mode):
 		59940: '60',
 		60000: '60'}
 
+codec_data = {
```

Comment:
Maybe use `from Components.Converter.PliExtraInfo import codec_data` instead of duplicating PliExtraInfo codec_data?